content/security/docs/incidents.md

```diff
@@ -82,7 +82,7 @@ An attacker may attempt to erase their misdeeds by terminating an affected node.
 This will serve as a warning to cluster administrators not to tamper with the affected Pods/Nodes until the investigation is complete. 
 
 ### Capture volatile artifacts on the worker node
-+ **Capture the operating system memory**. This will capture the Docker daemon and its subprocess per container.  [MargaritaShotgun](https://github.com/ThreatResponse/margaritashotgun), a remote memory acquisition tool, can aid in this effort. 
```

Comment:
This is old, but I think it still works.

Reply:
I tried it out a bit, and it seems to need some love. For example, I had to downgrade setuptools to be able to install it, and I also ran into some cryptic errors. I updated to link to the LiME/Volatility projects instead, which seem bigger and more updated (margaritashotgun was built around LiME), and I also linked to the AWS-made forensics solution.